packages/app-mobile/components/get-responsive-value.ts

```diff
@@ -0,0 +1,83 @@
+// getResponsiveValue() returns the corresponding value for
+// of a particular device screen width based on a valueMap argument
+// and breakpoints
+//
+// Breakpoints:
+// sm: < 481
+// md: >= 481
+// lg: >= 769px
+// xl: >= 1025px
+// xxl: >= 1201px
```

Comment:
Does that really work across devices these hardcoded values? For example on tablet, phone, but also Chromebook, and screens with hdpi?

Do you have a source for this?

Reply:
According to React Native's documentation, React Native uses density-independent pixels. That means the values still represent the same physical dimension across different devices.

> All dimensions in React Native are unitless, and represent density-independent pixels
[React Native Docs](https://reactnative.dev/docs/height-and-width)

Sources:
 [Device independent pixels](https://www.youtube.com/watch?v=zhszwkcay2A)
 [When is a pixel not a pixel](http://www.gregoryschmidt.ca/writing/when-is-a-pixel-not-a-pixel-pixels-points-dps)

The breakpoints are based on popular css responsive breakpoints.